### DISCUSSION TITLE
Two unrelated problems I just can't figure out

### DISCUSSION THREAD

**Discussion by dmmagic** [coteditor/CotEditor#1536] [Q&A]
Hi hello, I'm having a few problems that don't seem to have obvious solutions.

1) In Settings->Window there is a toggle for changing the background color of the line the cursor is in. I have it checked but the line does not change color. I've tried changing the current line color in Settings->Appearance, changing themes, creating themes, and duplicating existing themes--all to no effect.

2) CotEditor won't reopen windows from the last session. The option is toggled in Settings->General. I couldn't think of any way to troubleshoot this.

I also trashed and reinstalled the app. No joy. Any help would be greatly appreciated!

**Comment by 1024jp** [comment]
Thank you for the report. 

FYI: Next time, please report this kind of issue (potential bugs) not here but directly at [Issues](https://github.com/coteditor/CotEditor/issues) and separately issue by issue.

> 1. In Settings->Window there is a toggle for changing the background color of the line the cursor is in. I have it checked but the line does not change color. I've tried changing the current line color in Settings->Appearance, changing themes, creating themes, and duplicating existing themes--all to no effect.

This is definitely a bug, since CotEditor 4.6.0. It has already been reported at #1537 (though, it's in Japanese), and will be fixed in the next version. Thank you.

> 2. CotEditor won't reopen windows from the last session. The option is toggled in Settings->General. I couldn't think of any way to troubleshoot this.

Hmm, this can be a bug on the CotEditor side. However, I could not reproduce it so far. Could you create an issue at [Issues](https://github.com/coteditor/CotEditor/issues) and provide more information, such as your system version, CotEditor version, etc.
